### PR TITLE
Include PlanPal information in readme

### PR DESCRIPTION
**Pull request by brianchoon**
## Summary

This PR addresses issue #31 by removing the dummy placeholder information from the "About Us" section, and preparing it to reflect the actual context of the project.

## Changes
- Removed dummy text from the AboutUs component.
- Updated the AboutUs section to better reflect the project description and user profile.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
-**AddressBook is a desktop application for managing your contact details.** While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
+* **PlanPal** is an address book designed for student event planners at NUS who need to manage contacts (e.g., attendees, vendors, sponsors, and volunteers) for their events. 
+* PlanPal offers a centralized platform to organize, track, and access contact information, ensuring efficient coordination and smooth communication throughout the event planning process.
+* While it has a GUI, most of the user interactions happen using a CLI (Command Line Interface).
 
 * If you are interested in using AddressBook, head over to the [_Quick Start_ section of the **User Guide**](UserGuide.html#quick-start).
 * If you are interested about developing AddressBook, the [**Developer Guide**](DeveloperGuide.html) is a good place to start.
 
-
 **Acknowledgements**
```
Comment (chuajunyu) [inline docs/index.md:19]: Would it be good if you could add this acknowledgement according to what the CS2103T website requested too?

> A3.
> Acknowledge the original source of the code e.g.,
> This project is based on the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).

**Review by chuajunyu — CHANGES_REQUESTED** [review]
It would be good if you could add the extra acknowledgement in!

File: docs/index.md (hunk 1 above)
Comment (brianchoon) [reply docs/index.md:19]: Thanks Jun Yu for your suggestion, I agree that it is important and will implement the changes!

**Review by chuajunyu — APPROVED** [review]
LGTM!

**Pull request merged**